go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/firecracker-microvm/firectl
 
 require (
-	github.com/firecracker-microvm/firecracker-go-sdk v0.17.0
+	github.com/firecracker-microvm/firecracker-go-sdk v0.17.1-0.20191030150325-dbf9a1e05f090
```

Comment:
We should do a proper SDK release so we can reference something like "v0.19.0" in this go.mod file...

Reply:
v0.19.0 of the SDK is released. Can you update this PR to use it?

Reply:
Done, but have strange test failings.